src/App/App.js

```diff
@@ -28,19 +28,11 @@ class App extends Component {
     const formattedTitle = this.makeUpperCase(title);
     const movie = this.state.movies.find(movie => movie.title === formattedTitle);
     if (movie) {
-      this.setState({...this.state, selectedMovie: movie.id});
+      return movie.id;
     } else {
-      this.setState({ ...this.state, error: 'Sorry, looks like we can\'t find that title.' })
+      return false;
     }
   }
-
-  resetSelectedMovie = () => {
-    this.setState({...this.state, selectedMovie: ''});
-  }
-
-  resetError = () => {
-    this.setState({...this.setState, error: ''})
-  }
```

Comment:
This is from a previous fix I was trying, to reset the selected movie so the redirect wouldn't keep going to the movie that was selected last.